tldr.l

```diff
@@ -99,7 +99,7 @@ space [ \t]
 %{
   if (this.topState() === "title") {
     yytext = this.matches[1];
-    if (this.matches[1].match(/([^A-Za-z0-9_\. -])|(\.$)/)) yy.error(yylloc, 'TLDR013');
+    if (this.matches[1].match(/([^A-Za-z0-9_\+\. -])|(\.$)/)) yy.error(yylloc, 'TLDR013');
```

Comment:
IIRC, special characters don't need to be escaped within character ranges in regex. So both the `\+` and the `\.` could be changed to simply `+` and `.`.

Reply:
Done

Reply:
@waldyrious You're definitely right about that.  However, `\w` and `\d` still work inside them 🤔

Reply:
> `\w` and `\d` still work inside them 

You mean still _don't_ work, right?

Reply:
:smile: Actually, I mean _do_ work :

**Node.js**
```console
$ node
Welcome to Node.js v15.4.0.
Type ".help" for more information.
> /^[\d\w]$/.test('a')
true
> /^[\d\w]$/.test('4')
true
> /^[\d\w]$/.test('-')
false
> 
```

**Python**
```console
$ ipython
Python 3.9.1 (default, Dec 29 2020, 09:45:39) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.19.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import re

In [2]: re.match(r'^[\d\w]$', 'a')
Out[2]: <re.Match object; span=(0, 1), match='a'>

In [3]: re.match(r'^[\d\w]$', '4')
Out[3]: <re.Match object; span=(0, 1), match='4'>

In [4]: re.match(r'^[\d\w]$', '-')

In [5]:
```

Reply:
Oh cool, [TIL](https://www.regular-expressions.info/refshorthand.html)!

Reply:
Wait, I just realized that we could then replace `A-Za-z0-9_` with just `\w`!

Reply:
@waldyrious that site is fantastic! I wish the PDF were free :(